lib/pbench/test/unit/server/test_schema.py

```diff
@@ -92,20 +112,43 @@ class TestParameter:
     """
 
     def test_constructor(self):
+        """
+        Test the Parameter class constructor with various parameters.
+        """
+        v = Parameter("test", ParamType.LIST, element_type=ParamType.KEYWORD)
+        assert not v.required
+        assert v.name == "test"
+        assert v.type is ParamType.LIST
+        assert v.element_type is ParamType.KEYWORD
+        assert v.keywords is None
```

Comment:
Could you reverse the order of these two lines (ditto lines 129-130), so that all the assertions in all the scenarios in this test follow the same pattern (e.g., so that they match lines 136-137 et al.)?